java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -566,7 +580,7 @@ static NetworkDataframeMapper threeWindingTransformers() {
     }
 
     static NetworkDataframeMapper danglingLines() {
-        return NetworkDataframeMapperBuilder.ofStream(Network::getDanglingLineStream, getOrThrow(Network::getDanglingLine, "Dangling line"))
+        return NetworkDataframeMapperBuilder.ofStream(network -> network.getDanglingLineStream(), getOrThrow(Network::getDanglingLine, "Dangling line"))
```

Comment:
why reverting the Network::getDanglingLineStream ?

Reply:
The complier complains about signature ambiguity